### ISSUE TITLE
virtualenvwrapper.plugin.zsh does not support tmux and pyenv together

### ISSUE THREAD

**Issue by oskargicast** [ohmyzsh/ohmyzsh#4951]
Hi,

I started using pyenv with [pyenv-virtualenvwrapper](https://github.com/yyuu/pyenv-virtualenvwrapper) an it works like a sharm with this plugin.

The problem comes when I try to use tmux. For example, when I run this:

``` shell
$ tmux new -s session_name
```

I get `[exited]` as response:
![screenshot from 2016-03-27 19 16 05](https://cloud.githubusercontent.com/assets/1572530/14068672/77b70d74-f450-11e5-817b-b0d003606003.png)

I realized [this line](https://github.com/ohmyzsh/ohmyzsh/blob/HEAD@%7B2016-03-28T00:27:18Z%7D/plugins/virtualenvwrapper/virtualenvwrapper.plugin.zsh#L7) in the plugin makes tmux exits automatically:

``` shell
source ${${virtualenvwrapper}:c}
```

I could start tmux in my zsh config file. Despite that, I can not start any other session so the usage of tmuxinator, for instance, is not possible :(

I really appreciate your help. This is a [link with of my dotfiles](https://github.com/oskargicast/shascii).

Thanks.

**Comment by chngtrn** [comment]
Had the same issue. Had to remove virtualenvwrapper from plugins to be able to use pyenv.

**Comment by propyless** [comment]
Old issue, but I had this problem today and turns out it was due to asdf which  I use to manage different python version. The shims it creates wrappers for the binaries found in the python installations bin folder. So the shim looks like so.. and can't be sourced.

I just modified the oh-my-zsh virutalenvwrapper plugin with the paths to the actual virtualenvwrapper.sh script and that resolved the issue for me.

```
#!/usr/bin/env bash
# asdf-plugin: python 3.6.8
exec /home/propyless/.asdf/bin/asdf exec "virtualenvwrapper.sh" "$@"
```

Not sure if you have the same issue still today/using asdf or something similar, but this was the cause for me :)

**Comment by awgymer** [comment]
I think I may be having a similar issue. I am running on Mac Catalina 10.15.2.

I have installed `pyenv` via `homebrew` and then I used `pip` to install `virtualenvwrapper` using my `pyenv` python 3.8.2. I do not have `pyenv-virtualenv` or `pyenv-virtualenvwrapper` installed. 

When I added `virtualenvwrapper` to my plugins like so `plugins=(git pyenv virtualenvwrapper)` it caused my terminal to hang for a long time before giving me this:

```
Last login: Fri Apr 10 12:52:29 on ttys001
[Process completed]
```
